frontend/src/components/ConfigForm.vue

```diff
@@ -135,22 +158,32 @@
           <v-radio label="Direct Deploy" value="deploymentDirect"></v-radio>
         </v-radio-group>
 
-        <v-btn flat @click="appConfigStep = 2">Back</v-btn>
+        <v-btn flat @click="appConfigStep = 3">Back</v-btn>
 
         <v-dialog
           v-model="confirmationDialog"
           persistent
-          max-width="400"
-          v-if="userAppCfg.deploymentMethod === 'deploymentDirect'"
+          max-width="1200"
```

Comment:
Is there any way to make this be reactive and/or be defined as a constants variable elsewhere?

Reply:
max-width values in html is pretty standard, would set in the CSS if elsewhere